src/ddmd/backend/cgcod.c

```diff
@@ -882,7 +882,7 @@ void prolog(CodeBuilder& cdb)
                 farfunc ||
                 config.flags & CFGstack ||
                 xlocalsize >= 0x1000 ||
-                (usednteh & ~NTEHjmonitor) ||
+                (usednteh & (NTEH_try | NTEH_except | NTEHcpp | EHcleanup | EHtry | NTEHpassthru)) ||
```

Comment:
Looks like a good candidate for refactoring into a new flag, just like `NTEHcpp` is combination of several ones.

Reply:
There's a lot of refactoring that needs to be done. But one step at a time.